sdcm/cluster_k8s/minikube.py

```diff
@@ -141,35 +301,159 @@ def local_kubectl_version(self):  # pylint: disable=no-self-use
         #   Client Version: v1.18.5
         return LOCALRUNNER.run("kubectl version --client --short").stdout.rsplit(None, 1)[-1][1:]
 
+    def docker_pull(self, image):
+        LOGGER.info("Pull `%s' to Minikube' Docker environment", image)
+        self.remoter.run(f"docker pull -q {image}")
+
+    @property
+    def remoter(self) -> CommandRunner:
+        return self.host_node.remoter
+
+    def deploy_scylla_manager(self, pool_name: str = None) -> None:
+        self.deploy_minio_s3_backend()
+        super().deploy_scylla_manager(pool_name=pool_name)
+
+    @property
+    @abc.abstractmethod
+    def _create_kubectl_config_cmd(self):
+        pass
```

Comment:
You may want to use following python3 approach to reduce number of lines:

```
    @property
    @abc.abstractmethod
    def _create_kubectl_config_cmd(self): ...
```

Three dots is syntax sugar which allows us to skip usage of "pass" statement.

Reply:
fixed everywhere in the file